deepchem/lightning/tests/test_dc_lightning_module.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import deepchem as dc
+from deepchem.models import MultitaskClassifier
+import pytorch_lightning as pl
+import numpy as np
+import torch
+from torch.utils.data import DataLoader
+from deepchem.lightning.dc_lightning_module import DCLightningModule
+
+
+class MolnetDataset(torch.utils.data.Dataset):
```

Comment:
This doesn't have anything to do with molnet? Perhaps better to rename to `TestDataset`